driver/registry.go

```diff
@@ -197,6 +199,12 @@ func ReplaceTracer(f func(*otelx.Tracer) *otelx.Tracer) func(o *options) {
 	}
 }
 
+func Inspect(f func(reg Registry) error) func(o *options) {
```

Comment:
What is this used for? Just external usage?

Reply:
Exactly.